Automattic/wp-calypso
# update Site editor tracks e2e tests for new theme, gutenberg changes, and edge environment support.

#### Changes proposed in this Pull Request

Updates the site editor tracks e2e tests for proper GUTENBERG_EDGE environment support, as well as required updates for recent changes in Gutenberg and the theme for the test site.

#### Testing instructions



* Run this test suite locally and verify all tests path with the GUTENBERG_EDGE env variable in both `true` and `false` states.  ex - `env GUTENBERG_EDGE=true ./node_modules/.bin/mocha specs/specs-wpcom/wp-calypso-gutenberg-site-editor-tracking-spec.js`



Related to #
